packages/button/README.md

```diff
@@ -5,17 +5,19 @@ Mineral UI `Button` component. Find out more about [Mineral UI](https://github.c
 
 ## Installation
 
-1. Install project [peer dependencies](../../docs/peer-dependencies.md), if not already installed
-
-2. Install the package
+1. Install the package
 
   ```sh
   npm install --save @mineral-ui/button
   ```
 
+2. Install any missing peer dependencies reported by `npm` or `yarn`.
+
 
 ## Usage
 
+Wrap your app, or at least this branch of your app's render path in [ThemeProvider](https://github.com/mineral-ui/mineral-ui/docs/styling.md#themeprovider-theme) in order for the styles to apply correctly.
```

Comment:
I like that we're adding this note in each package's README. 👍 Please add a comma after 'path', i.e. "...render path, in..."